### ISSUE TITLE
Full Training Resource Requirements

### ISSUE THREAD

**Issue by zhw-zhang** [showlab/Kiwi-Edit#5]
Hi, thank you for good work. May I ask what resources are required for a complete training pipeline? For example, how many GPUs, what batch size, and how many days of training are needed?

**Comment by linyq17** [comment]
Thanks for your interest in our work! For reproduction, we recommend 32 GPUs with a global batch size of 128 (using gradient accumulation). Expect each training stage to take approximately 3–5 days.
